examples/separate-config/every-feature-multiple-dest-single-ms.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>jQuery multiselect plugin with two sides</title>
+    <meta charset="utf-8">
+
+    <link rel="icon" type="image/x-icon" href="https://github.com/favicon.ico" />
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
+    <link rel="stylesheet" href="../../css/style.css" />
+</head>
+<body>
+    <div id="wrap" class="container">
+        <h4 id="demo-zero-configuration-separate-js">Zero configuration, multiple destinations, separate JS file</h4>
+
+        <div class="row">
+            <div class="col-xs-5">
+                <select name="from[]" id="multi_d" class="form-control palette" size="26" multiple="multiple">
+                    <option value="1">C++</option>
+                    <option value="2">C#</option>
+                    <option value="3">Haskell</option>
+                    <option value="4">Java</option>
+                    <option value="5">JavaScript</option>
+                    <option value="6">Lisp</option>
+                    <option value="7">Lua</option>
+                    <option value="8">MATLAB</option>
+                    <option value="9">NewLISP</option>
+                    <option value="10">PHP</option>
+                    <option value="11">Perl</option>
+                    <option value="12">SQL</option>
+                    <option value="13">Unix shell</option>
+                </select>
+            </div>
+            
+            <div class="col-sm-2">
+                <button type="button" id="multi_d_undo" class="btn btn-primary btn-block">Undo</button>
+                <button type="button" id="multi_d_redo" class="btn btn-warning btn-block">Redo</button>
+                <hr style="margin: 40px 0 60px;" />
+                <button type="button" id="multi_d_rightAll_1" data-context="#multi_d_to_1" class="btn btn-default btn-block"><i class="glyphicon glyphicon-forward"></i></button>
```

Comment:
The buttons have the new data-context attribute, where the user provides which select destination should be associated with the button. This is only necessary when you use multiple destinations.